docs/source/release/v4.1.0/framework.rst

```diff
@@ -39,6 +39,7 @@ Improvements
 - Prevent an error due to the locale settings which may appear when reading, for instance, the incident energy Ei value from the logs in :ref:`ConvertUnits <algm-ConvertUnits>` and many other algorithms.
 - :code:`indices` and :code:`slicepoint` options have been added to :ref:`mantid.plots <mantid.plots>` to allow selection of which plane to plot from an MDHistoWorkspace. :code:`transpose` has also been added to transpose the axes of any 2D plot.
 - :ref:`Pseudo-Voigt <func-PseudoVoigt>` has been modified to be more in line with FULLPROF and GSAS.  One of its basic parameter, Height, is changed to Intensity.
+- :ref:`CalculatePolynomialBackground <algm-CalculatePolynomialBackground>` has an option to select the the minimizer used during fitting.
```

Comment:
I think here we should also add a note that it's more correct now due to passing parameters around with double precision.